test/single_file/initializers_outside_functions.expected

```diff
@@ -0,0 +1,14 @@
+int x = 1 + 2;
+
+namespace {
+  int y = x + x;
+
+  class A {
+    int z = y + y;
+    void foo() {
+      class B {
+        int w = !x;
+      };
+    }
+  };
+}
```

Comment:
The naming of this file appears to be wrong, it may also be missing the relevant .cc/.c file for testing.

Reply:
Fixed, thanks!